feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/extrinsic/RealExtrinsicService.kt

```diff
@@ -118,10 +120,14 @@ class RealExtrinsicService(
         extrinsicBuilder.formExtrinsic()
         val extrinsic = extrinsicBuilder.build()
 
-        return estimateFee(chain.id, extrinsic)
+        return estimateFee(chain.id, extrinsic).amount
     }
 
-    override suspend fun estimateFee(chainId: ChainId, extrinsic: String): BigInteger {
+    override suspend fun estimateFeeV2(chain: Chain, formExtrinsic: suspend ExtrinsicBuilder.() -> Unit): Fee {
```

Comment:
can we rename this function to estimateFee to overload it?

Reply:
We cannot. they will only differ by return type, which is not included into method signature so they will be considered same from overload resolution perspective